scripts/ci/test.sh

```diff
@@ -44,6 +44,8 @@ if [ -z "$ARGV_OPERATING_SYSTEM" ] || [ -z "$ARGV_ARCHITECTURE" ]; then
   usage
 fi
 
+make babel
```

Comment:
I'm not sure this is the right place for this, should it be added to the existing `make lint test ...` lines below?

Reply:
It could possibly as long as those are done sequentially, but I do think it's more clear that it happens before those by having it independently above. It's important that the linter and tests run after this.

Reply:
In that case this shouldn't be in the `test.sh` script, but elsewhere.